meta-resin-sumo/recipes-core/busybox/busybox_%.bbappend

```diff
@@ -0,0 +1,4 @@
+FILESEXTRAPATHS_prepend := "${THISDIR}/${PN}:"
+SRC_URI_append = " \
+	file://0001-Install-link-in-usr-bin-to-match-other-providers.patch \
```

Comment:
something smells funny about this patch.
What makes this a resinOS special? readonly-rootfs? I thought yocto had a few readonly rootfs things as well

Reply:
It's not resin specific. Is build system configuration specific. This is because we want to have the same path for the all the providers of that specific tool.